clang/include/clang/Frontend/FrontendOptions.h

```diff
@@ -408,6 +411,9 @@ class FrontendOptions {
   LLVM_PREFERRED_TYPE(bool)
   unsigned GenReducedBMI : 1;
 
+  /// Use Clang IR pipeline to emit code
+  unsigned UseClangIRPipeline : 1;
```

Comment:
```suggestion
  LLVM_PREFERRED_TYPE(bool)
  unsigned UseClangIRPipeline : 1;
```

Reply:
Ah I guess this is why we didn't have the `false` defualt before. This was added a few months ago @bcardosolopes